File: JS/Conditional Statements/WorldSwimmingRecord.js
function calcSwimmingTime(input){
    let record = Number(input[0]);
    let distance = Number(input[1]);
    let secFor1m = Number(input[2]);
    let sections15m = Math.floor(distance / 15);
    let time = distance * secFor1m + sections15m * 12.5;
    if (time < record){
        console.log(`Yes, he succeeded! The new world record is ${time.toFixed(2)} seconds.`)
    }
    else{
        console.log(`No, he failed! He was ${(time - record).toFixed(2)} seconds slower.`)
    }
}

// calcSwimmingTime(["55555.67", "3017", "5.03"])

File: Python Basics/exercises/world_swimming_record.py
import math

record_in_seconds = float(input())
distance = float(input())
seconds_per_m = float(input())

clean_time = (distance * seconds_per_m)
distance_15_meters = math.floor(distance / 15)
late = (distance_15_meters * 12.5)
total_time = (clean_time + late)
sec_to_record = abs(record_in_seconds - total_time)

if total_time < record_in_seconds:
    print(f" Yes, he succeeded! The new world record is {total_time:.2f} seconds.")
else:
    print(f"No, he failed! He was {sec_to_record:.2f} seconds slower.")